test/distributed/org/apache/cassandra/distributed/test/PartitionDenylistTest.java

```diff
@@ -58,9 +61,9 @@ public class PartitionDenylistTest extends TestBaseImpl
     public void checkStartupWithoutTriggeringUnavailable() throws IOException, InterruptedException, ExecutionException, TimeoutException
     {
         int nodeCount = 4;
-        System.setProperty("cassandra.ring_delay_ms", "5000"); // down from 30s default
-        System.setProperty("cassandra.consistent.rangemovement", "false");
-        System.setProperty("cassandra.consistent.simultaneousmoves.allow", "true");
+        RING_DELAY.setLong(5000);
```

Comment:
the comment "// down from 30s default" should be returned.

Reply:
Fixed.